packages/server/core/package.json

```diff
@@ -31,7 +31,7 @@
       "default": "./dist/cjs/index.js"
     },
     "./base": {
-      "types": "./base/index.ts",
+      "types": "./dist/types/base/index.d.ts",
```

Comment:
It should not be modified here，please see publishConfig

Reply:
Is just for source-code compiler?

Reply:
uni-builder use tsc to compiler, it seems not support source-code compile.